flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/FlinkConnectorOptions.java

```diff
@@ -69,6 +69,13 @@ public class FlinkConnectorOptions {
 
     public static final ConfigOption<Integer> SINK_PARALLELISM = FactoryUtil.SINK_PARALLELISM;
 
+    public static final ConfigOption<Boolean> SINK_SHUFFLE_BY_PARTITION =
+            ConfigOptions.key("sink.shuffle-by-partition.enable")
```

Comment:
If the name is the same as that of Flink, will it lead to another useless partition shuffle? See Flink `StreamPhysicalSinkRule`.

Reply:
Thanks @JingsongLi I have rebased from master and rename the config name